api/help/help-tidal-scale_de.php

```diff
@@ -0,0 +1,7 @@
+<iframe src="https://wiki.openseamap.org/wiki/De:Pegel" width="540" height="670" frameborder="0">
+  <p>Ihr Browser kann leider keine eingebetteten Frames anzeigen: 
+  Sie können die eingebettete Seite über den folgenden 
+  Verweis aufrufen: 
+  https://wiki.openseamap.org/index.php?title=De:Pegel&printable=yes
```

Comment:
This is the only link with &printable=yes... is this by intention?

Reply:
No idea it was like that before